src/backend/api/reviews.js

```diff
@@ -0,0 +1,86 @@
+const express = require("express");
+const reviewsRouter = express.Router();
+const dbcon = require("../database");
+
+// Returns all reviews.
+reviewsRouter.get("/", async (request, response) => {
+  let reviews = undefined;
+  try {
+    reviews = await dbcon("Review");
+  } catch (error) {
+    return response.status(500).send(error);
+  }
+  if (reviews.length < 1) {
+    return res.status(204).send();
+  }
+  response.send(reviews);
+});
+
+// Adds a new review to the database.
+reviewsRouter.post("/", async (request, response) => {
+  const review = request.body;
+  try {
+    await dbcon("Review").insert(review);
+  } catch (error) {
+    return response.status(500).send(error);
+  }
+  response.json({ success: true, message: "The review successfully added!" });
```

Comment:
This success message will show even if the request has been not successful. This line should go after line 23 if the request is successful.